Cargo.toml

```diff
@@ -20,3 +20,8 @@ serde_derive = "~1.0.3"
 default = ["std"]
 
 std = ["serde/std"]
+
+serde-collections = ["serde/collections"]
```

Comment:
Sure, fine to combine all under one feature, but then I would pick `collections` for consistency with serde and to give a hint as to specifically what is needed.